README.md

```diff
@@ -1106,3 +1106,10 @@ Check out the [createWith()](#createwith) method on how you can achieve this in
 
 > Tests marked as incomplete means they are either known issues or non-supported features,
 check included messages for more info.
+
+## Factories
+ 
+  > You can use default Laravel `factory()` helper for NeoEloquent models too.
+
+ - define needed factories inside `database/factories/`;
```

Comment:
add a reference to factories docs in Laravel's docs.

Reply:
ok